wordpress-mobile/WordPress-Android
# Api26/jobschedule reader post service

*Note:* Builds on top of #7603 which should be merged first. Otherwise this looks like a super big PR 😱 

Continues the API26 migration series. This time we bring `ReaderPostService` in its two versions, a `Service` for API below level 26, and `JobService` for API 26 and beyond.

To test:
1. Start the app
2. Go to the reader
3. Change tags filters, refresh, etc. and see that it works.

Use please both platform 26+ and below.

cc @nbradbury
